ctx_test.go

```diff
@@ -1247,6 +1247,28 @@ func Test_Ctx_Method(t *testing.T) {
 	utils.AssertEqual(t, MethodPost, c.Method())
 }
 
+// go test -run Test_Ctx_ClientHelloInfo
+func Test_Ctx_ClientHelloInfo(t *testing.T) {
+	t.Parallel()
+	app := New()
+	app.Post("/test", func(c *Ctx) error {
+		result := c.ClientHelloInfo()
+		if result != nil {
+			panic("Invalid ClientHelloInfo, it should be nil")
```

Comment:
Return an error with NewError instead of `panic()`. And we may need more tests i suppose

Reply:
I add more tests on this, please check it and tell me if it looks good for you 👍